beacon-chain/sync/subscriber_handlers.go

```diff
@@ -17,11 +20,22 @@ func (r *Service) voluntaryExitSubscriber(ctx context.Context, msg proto.Message
 }
 
 func (r *Service) attesterSlashingSubscriber(ctx context.Context, msg proto.Message) error {
-	// TODO(#3259): Requires handlers in operations service to be implemented.
+	as := msg.(*ethpb.AttesterSlashing)
```

Comment:
type assertion on `msg.(*ethpb.AttesterSlashing)` ?

defensive programming 101 😆

Reply:
i added it but then saw that the code that calls it is calling it by its type https://github.com/prysmaticlabs/prysm/blob/6d2a2ebadf984ec7c77e5b4450f4806148b92906/beacon-chain/p2p/gossip_topic_mappings.go#L12
i will add it anyway for resilience

Reply:
Correct. Hence defensive programming

Reply:
thanks